content/en/logs/log_collection/javascript.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Browser Log Collection
+kind: documentation
+aliases:
+  - /logs/log_collection/web_browser
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+- link: "logs/processing/parsing"
+  tag: "Documentation"
+  text: "Learn more about parsing"
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "logs/explorer/analytics"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "logs/faq/log-collection-troubleshooting-guide"
+  tag: "FAQ"
+  text: "Log Collection Troubleshooting Guide"
+---
+
+
+
+Send logs to Datadog from web browsers or other Javascript client thanks to the Datadog Client-side JavaScript logging library.
+
+With the `datadog-logs` library you are able to send log directory to Datadog from JS clients and leverage the following features:
+
+* Use the library as a logger. Everything is forwarded to Datadog as JSON documents.
+* Add `context` and extra attributes.
+* Forward every JavaScript errors.
+* Forward JavaScript's console logs.
+* Track real client IP address and useragent.
+* Automatic bulk posts.
+
+## Get the Public API Key
+
+**Public API Keys are in Private Beta** 
```

Comment:
how will they get access to the beta? a request to the support team?

Reply:
Yes request to support team.

Reply:
```suggestion
**Public API Keys are in Private Beta** Reach out to [Datadog support team](/help) to get it enabled for your Account.
```